### PR TITLE
Added Trichotillomania

### PR DESCRIPTION
**Pull request by dhruvapte26**
Solved #16 . I closed the previous PR due to conflicts in the code :sweat_smile:

### CODE REVIEW

**Comment by Megha66** [comment]
I m not able to send pull requests .. What should i do?

**Comment by icyflame** [comment]
Interesting entry. Please fix grammar, I would be happy to accept this. Also, your references need to be short ones. In #10 both your references were academic and very very long.

**Comment by Megha66** [comment]
Sir i said I m not able to send pull requests. What should i do with that

**Comment by dhruvapte26** [comment]
@icyflame , updated as per you told. :smile:

**Comment by Megha66** [comment]
Sir why aren't u responding to the pull requests?

**Comment by icyflame** [comment]
Hey @Megha66 , it is customary to wait at least 2 weeks before pinging a maintainer for the second time. Please be patient.

**Review by icyflame — CHANGES_REQUESTED** [review]
Change the paragraph to wrap at 70 characters.

**Comment by Megha66** [comment]
Sorry for the inconvenience

**Comment by dhruvapte26** [comment]
@icyflame , updated the `readme.md` as requested to reduce length

**Review by icyflame — CHANGES_REQUESTED** [review]
I am not sure if I was clear earlier:

> Change the paragraph to wrap at 70 characters.

I don't mind the length of the entry, that was fine earlier. I wanted you to wrap everyline of the paragraph at 70 characters so that it would look better in editors. Like [this](https://github.com/icyflame/awesome-social-science/blame/f0ef8b4017c8550855c23702d4ca906347264861/readme.md#L56-L59) or [this](https://github.com/icyflame/awesome-social-science/blame/f0ef8b4017c8550855c23702d4ca906347264861/readme.md#L89-L92)

File: readme.md [hunk 1]
```diff
@@ -176,6 +178,23 @@ the anecdote, you can throw away the data."
 
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
+## Pyschological Disorders
+
+## Trichotillomania
```
Comment (icyflame) [inline readme.md]: This should be `#### [Tricho...](link-to-wikipedia)`

**Review by icyflame — CHANGES_REQUESTED** [review]
Hey Dhruv, the changes are almost done now. Thanks! :smile:

File: readme.md (hunk 1 above)
Comment (dhruvapte26) [reply readme.md]: Done that :sweat_smile:

**Review by icyflame — CHANGES_REQUESTED** [review]
Fix conflicts and re open later

**Pull request closed without merging**